pretyping/nativenorm.ml

```diff
@@ -406,7 +406,7 @@ and nf_evar env sigma evk args =
     let fold accu d = EConstr.mkNamedProd_or_LetIn sigma d accu in
     let t = List.fold_left fold ty hyps in
     let t = EConstr.to_constr ~abort_on_undefined_evars:false sigma t in
-    let ty, args = nf_args env sigma args t in
+    let ty, args = nf_args env sigma (Array.to_list args) t in
```

Comment:
Can a floating-point number appear as an argument of an evar?

Reply:
I think this is already taken care of by the compilation of `MLarray` in https://github.com/coq/coq/pull/15081

Reply:
Right, I guess the native code itself never creates such an array, it goes through untouched. And my point is moot anyway, since this conversion is in the safe direction. If there is still a bug, it is necessarily elsewhere.